### PR TITLE
Move doc and config files in repo of analytics-zoo-doc to analytics-zoo 

### PR DESCRIPTION
**Pull request by helenlly**
compared with https://github.com/intel-analytics/analytics-zoo/pull/3321, update as below
1) Remove files that are not needed
2) Update the index.rst file
3) The project name should be "Analytics Zoo", not "analytics-zoo"
4) Rename doc to readthedocs, and source\readthedocs to source\doc`
5) update pyproject.toml ->license, describtion, project name

### CODE REVIEW

**Comment by helenlly** [comment]
test build can be seen in https://helentestdoc.readthedocs.io/en/latest/

File: readthedocs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+# The suffix of source filenames.
+from recommonmark.parser import CommonMarkParser
+source_suffix = {'.rst': 'restructuredtext',
+                 '.txt': 'markdown',
+                 '.md': 'markdown',}
+
+master_doc = 'index'
+
+project = 'Analytics Zoo'
+copyright = '2020, analytice-zoo'
```
Comment (jason-dai) [inline readthedocs/source/conf.py:50]: '2020, Analytics Zoo Authors'

File: readthedocs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+author = 'analytice-zoo'
```
Comment (jason-dai) [inline readthedocs/source/conf.py:51]: 'Analytics Zoo Authors'

File: readthedocs/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+# default: ``['localtoc.html', 'relations.html', 'sourcelink.html',
+# 'searchbox.html']``.
+#
+# html_sidebars = {}
+
+
+# -- Options for HTMLHelp output ---------------------------------------------
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'analytics-zoodoc'
```
Comment (jason-dai) [inline readthedocs/source/conf.py:141]: 'Analytics Zoo Documentation'?

File: readthedocs/source/doc/Orca/Overview/orca-context.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Orca Context
```
Comment (jason-dai) [inline readthedocs/source/doc/Orca/Overview/orca-context.md:1]: please update with the latest file

File: readthedocs/source/conf.py (hunk 1 above)
Comment (helenlly) [reply readthedocs/source/conf.py]: done

File: readthedocs/source/conf.py (hunk 2 above)
Comment (helenlly) [reply readthedocs/source/conf.py]: done

File: readthedocs/source/conf.py (hunk 3 above)
Comment (helenlly) [reply readthedocs/source/conf.py]: done

File: readthedocs/source/doc/Orca/Overview/orca-context.md (hunk 4 above)
Comment (helenlly) [reply readthedocs/source/doc/Orca/Overview/orca-context.md:1]: done

**Pull request merged**